src/screens/Server.js

```diff
@@ -203,6 +203,7 @@ class Server extends React.Component {
   confirmTestnetServer = () => {
     this.props.wallet.changeServer(this.state.selectedServer);
     if (this.props.useWalletService) {
+      this.props.wallet.changeWsServer(this.state.selectedWsServer);
```

Comment:
This method here `changeWsServer` already sets the ws URL in the config, right? So we can remove the line below?

Reply:
Just tested and yes, thanks

Done in 6a459cb8abaf7245c0e76d816f84cd96b080e76c